graql/reasoner/query/ReasonerQueryImpl.java

```diff
@@ -262,6 +280,13 @@ public void checkValid() {
         return pattern;
     }
 
+    @Override
+    public Pattern getPattern(Map<Variable, Concept> map){
```

Comment:
Look at moving into the supertype

Reply:
The implementation of `public Pattern getPattern(Map<Variable, Concept> map)` across `CompositeQuery`, `DisjunctiveQuery` and `ReasonerQueryImpl` is nearly identical, but to make them the same would nest conjunctions in conjunctions, so leaving them as-is for now